docs: Added documentation and cross-link for Azure site extension (#18896)
newrelic/docs-website: src/content/docs/apm/agents/nodejs-agent/getting-started/monitor-your-nodejs-app.mdx

    If you have a preference for manual installation via the command line, we currently have the option for [Legacy instrumentation for Lambda monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/enable-serverless-monitoring-aws-lambda-legacy/#node).
  </Collapser>

  <Collapser
      id="azure"
      title="Azure web app on Windows"
  >

>     If you'd like to instrument an Azure Web App (provided that it's on Windows and deployed as code), you can easily install and configure the New Relic Node Agent with the [Azure Node Agent Site Extension](/docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure).

Reviewer: ```suggestion
    If you'd like to instrument an Azure Web App for your apps deployed as code in a Windows environment, you can easily install and configure the New Relic Node Agent with the [Azure Node Agent Site Extension](/docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure). 
```

>     If you'd like to instrument an Azure Web App for your apps deployed as code in a Windows environment, you can easily install and configure the New Relic Node Agent with the [Azure Node Agent Site Extension](/docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure). 

Reviewer: ```suggestion
    If you'd like to instrument an Azure web app for your apps deployed as code in a Windows environment, you can easily install and configure the New Relic Node Agent with the [Azure Node Agent Site Extension](/docs/agents/nodejs-agent/hosting-services/nodejs-agent-microsoft-azure). 
```

Reviewer: not sure if we're referring to azure web app in the general, like any web app, or as a specific product Azure offers "Azure Web App"
